Open Judge System/Data/OJS.Data.Models/Problem.cs

```diff
@@ -19,6 +19,10 @@ public class Problem : DeletableEntity, IOrderable
 
         public virtual Contest Contest { get; set; }
```

Comment:
Do we have a case. that one problem will have both Problem group and Contest?

Reply:
For this issue we are keeping this connection, will be refactored further more as part of the bigger issue for this problem https://github.com/SoftUni-Internal/suls-issues/issues/3777